static/app/views/insights/common/components/widgets/overviewSlowAssetsWidget.tsx

```diff
@@ -0,0 +1,191 @@
+
+  const resourceQuery = getResourcesEventViewQuery({}, DEFAULT_RESOURCE_TYPES).join(' ');
+  const search = new MutableSearch(`has:span.group ${resourceQuery} ${query}`);
+  const referrer = Referrer.ASSETS_BY_TIME_SPENT;
+  const groupBy = SpanFields.NORMALIZED_DESCRIPTION;
+  const yAxes = 'sum(span.self_time)';
+  const totalTimeField = 'sum(span.self_time)';
+
+  const {
+    data: assetListData,
+    isLoading: isAssetListLoading,
+    error: assetListError,
+  } = useSpans(
+    {
+      fields: [
+        'span.group',
+        'project.id',
+        'sentry.normalized_description',
+        totalTimeField,
+      ],
+      sorts: [{field: totalTimeField, kind: 'desc'}],
+      search,
+      limit: 3,
+      noPagination: true,
+    },
+    referrer
+  );
+
+  const {
+    data: assetSeriesData,
+    isLoading: isAssetSeriesLoading,
+    error: assetSeriesError,
+  } = useTopNSpanSeries(
+    {
+      search: `span.group:[${assetListData?.map(item => `"${item['span.group']}"`).join(',')}]`,
+      fields: [groupBy, yAxes],
+      yAxis: [yAxes],
+      sort: {field: yAxes, kind: 'desc'},
+      topN: 3,
+      enabled: assetListData?.length > 0,
+    },
+    referrer
+  );
+
+  const isLoading = isAssetSeriesLoading || isAssetListLoading;
+  const error = assetSeriesError || assetListError;
+
+  const hasData = assetListData && assetListData.length > 0 && assetSeriesData.length > 0;
+
+  const colorPalette = theme.chart.getColorPalette(assetSeriesData.length - 1);
+
+  const visualization = (
+    <WidgetVisualizationStates
+      isEmpty={!hasData}
+      isLoading={isLoading}
+      error={error}
+      emptyMessage={<WidgetEmptyStateWarning />}
+      VisualizationType={TimeSeriesWidgetVisualization}
+      visualizationProps={{
+        id: 'overviewSlowAssetsWidget',
+        showLegend: props.loaderSource === 'releases-drawer' ? 'auto' : 'never',
+        plottables: assetSeriesData.map(
+          (ts, index) =>
+            new Line(convertSeriesToTimeseries(ts), {
+              color: colorPalette[index],
+            })
+        ),
+        ...props,
+        ...releaseBubbleProps,
+      }}
+    />
+  );
+
+  const footer = hasData && (
+    <WidgetFooterTable>
+      {assetListData?.map((item, index) => (
+        <Fragment
+          key={`${item['project.id']}-${item['span.group']}-${item['sentry.normalized_description']}`}
+        >
+          <div>
+            <SeriesColorIndicator
+              style={{
+                backgroundColor: colorPalette[index],
+              }}
+            />
+          </div>
+          <div>
+            <SpanDescriptionCell
+              projectId={Number(item['project.id'])}
+              group={item['span.group']}
+              description={item['sentry.normalized_description']}
+              moduleName={ModuleName.RESOURCE}
+            />
+          </div>
+          <span>{getDuration((item[totalTimeField] ?? 0) / 1000, 2, true)}</span>
```

Comment:
I believe we have a `Duration` component at `static/app/components/duration.tsx`, would we be able to use it here?